src/azure-cli/azure/cli/command_modules/appservice/resources/LinuxFunctionsStacks.json

```diff
@@ -259,6 +259,40 @@
                 "isDeprecated": null
             }
         },
+        {
+            "id": null,
+            "name": "powershell",
+            "type": "Microsoft.Web/availableStacks?osTypeSelected=LinuxFunctions",
+            "properties": {
+                "name": "powershell",
+                "display": "PowerShell Core",
+                "dependency": null,
+                "majorVersions": [
+                    {
+                        "displayVersion": "7.0",
+                        "runtimeVersion": "PowerShell|7",
+                        "supportedFunctionsExtensionVersions": [
+                            "~3"
+                        ],
+                        "isDefault": true,
+                        "minorVersions": [],
+                        "applicationInsights": true,
+                        "appSettingsDictionary": {
+                            "FUNCTIONS_WORKER_RUNTIME": "powershell"
+                        },
+                        "siteConfigPropertiesDictionary": {
+                            "use32BitWorkerProcess": false,
+                            "linuxFxVersion": "PowerShell|7"
+                        },
+                        "isPreview": false,
+                        "isDeprecated": false,
+                        "isHidden": false
```

Comment:
Please use: https://github.com/pragnagopa/azure-functions-supported-runtime-stacks/blob/master/LinuxFunctionsStacks.json

We want this stack to be hidden until we roll out functions host changes to support linux consumption